Docs/Book: Unexpected unicode character makes pdflatex build fail

**Describe the bug**

Trying to build the PDF, I get:

```
[90]

! Package inputenc Error: Unicode character − (U+2212)
(inputenc)                not set up for use with LaTeX.

See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.5767 ...n, D. “Algorithmic Analysis of Cahn−
                                                  Ingold−Prelog Rules of S...

? 
! Emergency stop.
 ...                                              
                                                  
l.5767 ...n, D. “Algorithmic Analysis of Cahn−
                                                  Ingold−Prelog Rules of S...

!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on RDKit.log.
Latexmk: Index file 'RDKit.idx' was written
Latexmk: Summary of warnings from last run of *latex:
  =====Latex reported missing or unavailable character(s).
=====See log file for details.
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'RDKit.log' for details
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs, or warnings treated as errors.
Latexmk: Examining 'RDKit.log'
=== TeX engine is 'pdfTeX'
Latexmk: Errors, so I did not complete making targets
make[3]: *** [Makefile:29: RDKit.pdf] Error 12
```
This is due to the em-dash style hyphen in line 738 of `Docs/Book/RDKit_Book.rst` introduced in 19bdd21de1c90f01f630b28ff72b3af9176d886f

```
.. [#newcip]  Hanson, R. M., Musacchio, S., Mayfield, J. W., Vainio, M. J., Yerin, A., Redkin, D. "Algorithmic Analysis of Cahn−Ingold−Prelog Rules of Stereochemistry: Proposals for Revised Rules and a Guide for Machine Implementation." J. Chem. Inf. Model. 2018, 58, 1755-1765.
```
Replacing the two `−` in "Cahn-Ingold-Prelog" with `-` makes the build go fine.

**To Reproduce**

Run `cd Docs/Book; RDBASE=$(CURDIR) $(MAKE) latexpdf`

**Expected behavior**

PDF builds fine.

**Configuration (please complete the following information):**
 - RDKit version: 202009.1
 - OS: Debian unstable

--- a/Code/GraphMol/CIPLabeler/CIPLabeler.h
+++ b/Code/GraphMol/CIPLabeler/CIPLabeler.h
@@ -29,7 +29,7 @@ namespace CIPLabeler {
  * described in:
  *
  * Hanson, R. M., Musacchio, S., Mayfield, J. W., Vainio, M. J., Yerin, A.,
- * Redkin, D. Algorithmic Analysis of Cahn−Ingold−Prelog Rules of
+ * Redkin, D. Algorithmic Analysis of Cahn--Ingold--Prelog Rules of
  * Stereochemistry: Proposals for Revised Rules and a Guide for Machine
  * Implementation. J. Chem. Inf. Model. 2018, 58, 1755-1765.
  *

--- a/Code/GraphMol/CIPLabeler/Wrap/rdCIPLabeler.cpp
+++ b/Code/GraphMol/CIPLabeler/Wrap/rdCIPLabeler.cpp
@@ -42,7 +42,7 @@ BOOST_PYTHON_MODULE(rdCIPLabeler) {
       "of https://github.com/SiMolecule/centres, which was originally "
       "written by John Mayfield. The original algorithm is described in:\n\n"
       "Hanson, R. M., Musacchio, S., Mayfield, J. W., Vainio, M. J., Yerin, "
-      "A., Redkin, D.\nAlgorithmic Analysis of Cahn−Ingold−Prelog Rules of "
+      "A., Redkin, D.\nAlgorithmic Analysis of Cahn--Ingold--Prelog Rules of "
       "Stereochemistry:\nProposals for Revised Rules and a Guide for Machine "
       "Implementation.\nJ. Chem. Inf. Model. 2018, 58, 1755-1765.\n";
 

--- a/Docs/Book/RDKit_Book.rst
+++ b/Docs/Book/RDKit_Book.rst
@@ -1735,7 +1735,7 @@ type definitions.
 .. [#morganFP] http://pubs.acs.org/doi/abs/10.1021/ci100050t
 .. [#gobbiFeats] https://doi.org/10.1002/(SICI)1097-0290(199824)61:1%3C47::AID-BIT9%3E3.0.CO;2-Z
 .. [#labutecip] Labute, P. "An Efficient Algorithm for the Determination of Topological RS Chirality" Journal of the Chemical Computing Group (1996)
-.. [#newcip]  Hanson, R. M., Musacchio, S., Mayfield, J. W., Vainio, M. J., Yerin, A., Redkin, D. "Algorithmic Analysis of Cahn−Ingold−Prelog Rules of Stereochemistry: Proposals for Revised Rules and a Guide for Machine Implementation." J. Chem. Inf. Model. 2018, 58, 1755-1765.
+.. [#newcip]  Hanson, R. M., Musacchio, S., Mayfield, J. W., Vainio, M. J., Yerin, A., Redkin, D. "Algorithmic Analysis of Cahn--Ingold--Prelog Rules of Stereochemistry: Proposals for Revised Rules and a Guide for Machine Implementation." J. Chem. Inf. Model. 2018, 58, 1755-1765.
 .. [#nadinecanon] Schneider, N., Sayle, R. A. & Landrum, G. A. Get Your Atoms in Order-An Open-Source Implementation of a Novel and Robust Molecular Canonicalization Algorithm. J. Chem. Inf. Model. 2015, 55, 2111-2120.
 .. [#eitherend] It's ok to have two identically ranked atoms on the two ends of the bond, but having two identically ranked atoms on the same end indicates that it's not a potential stereobond.
 

--- a/Docs/Code/CIPLabeling.md
+++ b/Docs/Code/CIPLabeling.md
@@ -5,7 +5,7 @@ originally written by John Mayfield in Java. The original algorithm was
 described in:
 
 Hanson, R. M., Musacchio, S., Mayfield, J. W., Vainio, M. J., Yerin, A.,
-Redkin, D. Algorithmic Analysis of Cahn−Ingold−Prelog Rules of
+Redkin, D. Algorithmic Analysis of Cahn--Ingold--Prelog Rules of
 Stereochemistry: Proposals for Revised Rules and a Guide for Machine
 Implementation. J. Chem. Inf. Model. 2018, 58, 1755-1765.
 
@@ -16,7 +16,7 @@ The main function is C++ RDKit:: CIPLabeler::assignCIPLabels()/ Python:
 rdkit.Chem.rdCIPLabeler.AssignCIPLabels().
 
 `assignCIPLabels()` calculates E/Z bond stereochemistry, R/S tetrahedral
-chirality, and r/s pseudochirality according to the Cahn−Ingold−Prelog rules.
+chirality, and r/s pseudochirality according to the Cahn--Ingold--Prelog rules.
 Each is stored in the _CIPCode property - including for bonds.
 
 assignCIPLabels() relies on the caller to mark potential stereocenters.

